controllers/flinkcluster_updater.go

```diff
@@ -239,6 +245,22 @@ func (updater *_ClusterStatusUpdater) deriveClusterStatus() flinkoperatorv1alpha
 				}
 			}
 		}
+
+		// If ingress does not have host, get ip from status
+		if len(urls) == 0 && observedJmIngress.Status.LoadBalancer.Ingress != nil {
+			var scheme string
+			if useTLS {
+				scheme = "https://"
+			} else {
+				scheme = "http://"
+			}
+			for _, ingress := range observedJmIngress.Status.LoadBalancer.Ingress {
+				if ingress.IP != "" {
```

Comment:
Should we add host as well if it is present?

Reply:
Yes, all of the observed information looks good to include.